biojava-structure/src/main/java/org/biojava/nbio/structure/align/fatcat/calc/FatCatParameters.java

```diff
@@ -455,4 +456,11 @@ private Object  getValue(String name){
 
 	}
 
+	public boolean isOptimizeAlignment() {
+		return optimizeAlignment;
+	}
+
+	public void setOptimizeAlignment(boolean optimizeAlignment) {
+		this.optimizeAlignment = optimizeAlignment;
```

Comment:
Can you copy the documentation for the parameter here? Users can read it here. (same here)